astropy/coordinates/tests/test_separation.py

```diff
@@ -351,3 +355,54 @@ def test_return_types(coord_class, method, output_type):
     """
     coord = coord_class(0 * u.deg, 0 * u.deg, 1 * u.pc)
     assert type(getattr(coord, method)(coord)) is output_type
+
+
+@pytest.mark.parametrize("coord_class", [SkyCoord, ICRS])
+@pytest.mark.parametrize(
+    "frame_mismatch_kwarg,expectation",
```

Comment:
extremely minor but while we're at it
```suggestion
    "frame_mismatch_kwarg, expectation",
```

Reply:
Ruff has a few rules for enforcing a consistent style in fixtures and parametrizations, including [PT006](https://docs.astral.sh/ruff/rules/pytest-parametrize-names-wrong-type/). I brought these rules up in the developer telecon last September, but the overall opinion was that enforcing them is not worth the effort, so currently contributors are free to use whatever style they like, as long as `pytest` can parse it.

I use comma separated values (not comma-and-space separated values) because that is what the examples in `pytest` documentation use.

Reply:
fair enough !